### DISCUSSION TITLE
Add `output_path` parameter to `frameit`

### DISCUSSION THREAD

**Discussion by mollyIV** [fastlane/fastlane#16895] [Ideas]
[`frameit`](https://docs.fastlane.tools/actions/frameit/) tool assumes that the `fastlane/screenshots` directory is preferred. Currently there's no way of specifying where the framed screenshots should be placed. Being able to point to a directory where to put the framed screenshots could be helpful:

```ruby
lane :screenshots do
  capture_screenshots

  screenshots = lane_context[SharedValues::SNAPSHOT_SCREENSHOTS_PATH]
  framed_screenshots = File.join(Dir.pwd, "framed_screenshots")
  frame_screenshots(
      path: screenshots,
      output_path: framed_screenshots
  )

  deliver(
    screenshots_path: framed_screenshots
    ...
  )
end
```
I'd be happy to implement this feature 😊 

Please let me know what you think 🙇

**Comment by antranapp** [comment]
I am new to frameit and was looking for this option as well. It'd be a great addition.

I wish I could contribute to this as well, but my Ruby knowledge is = 0
